docs/tutorials/nb_shap_feature_elimination.ipynb

```diff
@@ -293,14 +294,15 @@
    "source": [
     "## Set up the model and model tuning\n",
     "\n",
-    "You need to set up the model and it's hyperparameter optimization space/grid for the Backwards Feature Elimination. `Probatus` requires a **tree-based binary classifier** in order to speed up the computation of SHAP feature importance at each step. \n",
+    "You need to set up the model that you would like to use in the feature elimination. `Probatus` requires a **tree-based binary classifier** in order to speed up the computation of SHAP feature importance at each step. \n",
```

Comment:
Minor detail, but I see you capitalize probatus. Is it `Probatus` or `probatus`? I recommend lower-case.

Reply:
Another thought. If you use `KernelExplainer` instead of `TreeExplainer`, you get support for any model. Perhaps in a future version we could let the use specify which they would like to use. I would rephrase "probatus requires a tree-based.."  to something like "probatus currently only supports tree-based binary classifiers"

Reply:
Done, used lower-case